tests/cupy_tests/binary_tests/test_packing.py

```diff
@@ -39,3 +40,17 @@ def test_unpackbits(self):
         self.check_unpackbits([1])
         self.check_unpackbits([255])
         self.check_unpackbits([100, 200, 123, 213])
+        a = cupy.array([10, 20, 30])
+        pytest.raises(TypeError, cupy.unpackbits, a)
+        pytest.raises(TypeError, cupy.unpackbits, a.astype(float))
+
+    def test_pack_unpack_order(self):
+        for bo in ['big', 'little']:
+            self.check_unpackbits([], bitorder=bo)
+            self.check_unpackbits([0], bitorder=bo)
+            self.check_unpackbits([1], bitorder=bo)
+            self.check_unpackbits([255], bitorder=bo)
+            self.check_unpackbits([100, 200, 123, 213], bitorder=bo)
```

Comment:
seems like check is not asserting that the bits where correctly unpacked. Can you add a check that verifies that the unpacking was effectively done in big or little endian?

Reply:
Here we're checking against numpy counterpart implementation following `test_unpackbits`
pattern. So if 'big' endian from cupy matches that of numpy then we're doing a good job.
Should I not test against numpy in the case of bitorder ?

Reply:
ah sorry, i did not properly read the check code, then this is fine!!